### PR TITLE
Make collection map listing tab permission consistent with admin page

### LINKED ISSUE #50: Implement permission for collections with maps admin sub-tab?
I was updating documentation and I realize that if you have access to the Islandora Simple Maps admin interface you can disable anyone's collection map. 

I then realized that I had half implemented this by specifying a non-existent permission to that tab.
https://github.com/mjordan/islandora_simple_map/blob/7.x/islandora_simple_map.module#L45

This task is to either change that permission to be `"administer site configuration"` like the others or implement the other permission?

@mjordan?

### PR DESCRIPTION
**Pull request by whikloj**
Resolves #50

### CODE REVIEW

File: islandora_simple_map.module [hunk 1]
```diff
@@ -42,7 +42,7 @@ function islandora_simple_map_menu() {
     'title' => 'Collection Maps',
     'description' => 'Collections displaying a map of all member points',
     'page callback' => 'drupal_get_form',
-    'page arguments' => array('islandora_simple_map_admin_collection_display'),
+    'page arguments' => array('administer site configuration'),
```
Comment (mjordan) [inline islandora_simple_map.module]: Shouldn't that be 'access arguments'?

**Comment by whikloj** [comment]
OMG 🤦‍♂️

Okay so I was [calling](https://github.com/uml-digitalinitiatives/islandora_simple_map/blob/7f365ee3e552f1751cf357f029ab51c4df8ec2dd/islandora_simple_map.module#L46)  a [custom access function](https://github.com/uml-digitalinitiatives/islandora_simple_map/blob/7f365ee3e552f1751cf357f029ab51c4df8ec2dd/islandora_simple_map.module#L96-L99) that checked if you had the correct permissions ('administer site configuration') and that you had enabled Collection maps. 

This PR is not needed, maybe a better option is 1. for me to slow down and 2. to pass the access argument into my custom function to make it clearer the permissions needed.

Don't merge this, I'll do that first.

**Comment by whikloj** [comment]
@mjordan this is ready now or it can be closed. It has not actual impact on anything.

**Comment by whikloj** [comment]
I'm closing this, its not needed and probably just makes the code worse.

**Pull request closed without merging**